### PR TITLE
Adding an internal guide on bug tickets

### PR DESCRIPTION
**Pull request by amyblais**
Documenting slides that I created: https://docs.google.com/presentation/d/1Cp6A_yo7LgUmU2wnH1Pk56-AgkfS9Jwqg9LrP-ndvcY/edit#slide=id.p.

This guide should be shared with all R&D.

### CODE REVIEW

File: source/process/training.rst [hunk 1]
```diff
@@ -767,3 +767,37 @@ This is restricted to people with an `@mattermost.com` email address.  It is mea
 Partners team
 ---------------------------------------------------------
 This is an invite only team meant for Mattermost staff and partners.  This is a team for staff to interact with partners and for partners to interact with other partners. You should prefer public channels when you want partner-to-partner interactions and private channels when you want staff-to-partner interactions.  Remember that any partner added to the team will be able to join any public channel.
+
+---------------------------------------------------------
+Creating New Jira Bug Tickets
+---------------------------------------------------------
+
+A bug is any “obvious error”. If you’re not reporting an obvious error, please file a Story ticket instead. Errors on `unsupported platforms <https://docs.mattermost.com/install/requirements.html>`_ are not considered bugs.
+
```
Comment (wiersgallak) [inline source/process/training.rst]: Maybe we can be more explicit on obvious error to how the product or feature is functioning or any obvious UI errors. 0/5

File: source/process/training.rst [hunk 2]
```diff
@@ -767,3 +767,37 @@ This is restricted to people with an `@mattermost.com` email address.  It is mea
...
+1. Confirm you’re filing a new issue
+---------------------------------------------------------
+
+ - Search existing tickets in Jira to confirm your issue isn’t already filed by someone else.
```
Comment (lindalumitchell) [inline source/process/training.rst]: Will people need more guidance about how/where to search in JIRA? Could say something such as `in the Mattermost project`, but then that does risk making the doc more brittle and at risk of becoming out of date, even without a specific URL (which was my first thought).

File: source/process/training.rst [hunk 3]
```diff
@@ -767,3 +767,37 @@ This is restricted to people with an `@mattermost.com` email address.  It is mea
...
+
+2. Confirm the bug is not a security issue
+---------------------------------------------------------
+
+ - If your issue involves security, please mark the bug ticket as internal.
```
Comment (lindalumitchell) [inline source/process/training.rst]: Add a phrase after "involves security" such as "or includes confidential information" or "or includes customer information"?

File: source/process/training.rst [hunk 4]
```diff
@@ -767,3 +767,37 @@ This is restricted to people with an `@mattermost.com` email address.  It is mea
...
+
+3. Information needed on Mattermost bug tickets
+---------------------------------------------------------
+
+ - **Labels:** Add a ``customer-bug`` label if the ticket is based on a customer or a community bug report.
+ - **Environment:** Mattermost server and version, OS and version, Mattermost mobile app version, Mattermost desktop app version (include the ones that are relevant).
```
Comment (lindalumitchell) [inline source/process/training.rst]: Should we add to Environment (or in a separate item) something about any notable Mattermost configurations (such as HA, Elasticsearch, image proxy, SSO)—anything that might be relevant in helping narrow down the issue?

**Pull request merged**